ls1intum/Artemis
# Feature/questions answers/markdown

### Checklist

### Motivation and Context


Students, tutors and instructors can only use plain text to write questions and answers limiting their ability to describe the problem or solution precisely.
The buttons of the Markdown editor take too much space and don't look good on smaller screens.

### Description

I added the Markdown editor to the questions and answers component.
I changed the design of the buttons in the markdown editor UI to be smaller in order to fit better on smaller screens / spaces.

**NOTE: If you encounter any bugs with the markdown editor please let me know. I will be further working on it in a separate PR, in which I will also address the functionality.**
(Also if you have any wishes or suggestions for the Markdown editor)

### Steps for Testing


1. Log in to Artemis
(1a. Enable questions and answers on a course if deactivated) 
2. Navigate to a lecture or exercise
3. Markdown should now be available for questions and answers
4. View any other Markdown editor and check if buttons are displayed correctly

### Screenshots


Markdown editor for question (with smaller buttons):
<img width="419" alt="Bildschirmfoto 2020-04-20 um 19 04 16" src="https://user-images.githubusercontent.com/51987021/79779754-2547e800-833b-11ea-9bbb-01e980057979.png">

Editor after changes:
<img width="417" alt="Bildschirmfoto 2020-04-20 um 22 59 45" src="https://user-images.githubusercontent.com/51987021/79799455-c3977600-835a-11ea-8f91-0ec5ce558083.png">
